poc/tests/test_vdaf_ping_pong.py

```diff
@@ -236,43 +237,58 @@ def test_multi_round(self) -> None:
                 rand,
             )
 
-            (leader_state, msg) = vdaf.ping_pong_leader_init(
+            leader_init_state = vdaf.ping_pong_leader_init(
```

Comment:
By declaring this variable to be of a broader type, we can skip using a separate variable name, and get rid of the cast below.
```suggestion
            leader_state: State = vdaf.ping_pong_leader_init(
```

Reply:
Excellent tip, thank you!

Reply:
Done.